packages/atst/src/index.ts

```diff
@@ -0,0 +1,14 @@
+import { writeContract } from '@wagmi/core'
+export { prepeareWriteAttestation } from './lib/prepareWriteAttestation'
+export { readAttestation } from './lib/readAttestation'
+
+export { abi } from './lib/abi'
+
+/**
+ * Will type this properly later
+ *
+ * @example
+ * const config = await prepareAttestation(about, key, value)
+ * const tx = await writeAttestation(config)
+ */
+export const writeAttestation = writeContract
```

Comment:
To write an attestation you just prepare one and then pass it into writeContract.

This is liable to greatly change before release

Reply:
This isn't changing.  The reasoning is we are doing the minimum viable attestation station and we only carefuly add extra features when users request them